robolectric/src/test/java/org/robolectric/shadows/ShadowAccessibilityManagerTest.java

```diff
@@ -126,4 +130,67 @@ public void getSentAccessibilityEvents_returnsAllSentAccessibilityEventsInOrder(
         .containsExactly(event1, event2, event3)
         .inOrder();
   }
+
+  private static AccessibilityManager.AccessibilityStateChangeListener
+      createSimpleAccessibilityStateChangeListener() {
+    return new AccessibilityStateChangeListener() {
+      @Override
+      public void onAccessibilityStateChanged(boolean b) {}
+    };
+  }
+
+  @Test
+  public void addAccessibilityStateChangeListener_shouldAddListener() {
+    AccessibilityManager.AccessibilityStateChangeListener listener1 =
+        spy(createSimpleAccessibilityStateChangeListener());
+    AccessibilityManager.AccessibilityStateChangeListener listener2 =
+        spy(createSimpleAccessibilityStateChangeListener());
+    accessibilityManager.addAccessibilityStateChangeListener(listener1);
+    accessibilityManager.addAccessibilityStateChangeListener(listener2);
+
+    shadowOf(accessibilityManager).setEnabled(true);
+
+    verify(listener1).onAccessibilityStateChanged(true);
```

Comment:
What about adding `getEnabled` method to custom listener, and using it instead of `Mockito#spy` to check state changed?

Reply:
That works for me - I saw usages of `spy` elsewhere so I thought I'd try to follow conventions, I much prefer your suggestion. Will update this PR!